utils/call-gen/src/lib.rs

```diff
@@ -32,6 +32,8 @@ pub use send_message::SendMessageArgs;
 pub use upload_code::UploadCodeArgs;
 pub use upload_program::UploadProgramArgs;
 
+pub type GearProgGenConfig = gear_wasm_gen::GearConfig;
```

Comment:
```suggestion
pub(crate) type GearProgGenConfig = gear_wasm_gen::GearConfig;
```

Reply:
it's exported to other crates with this name